### PR TITLE
Traveller profile

### PR DESCRIPTION
**Pull request by zeus999**

### CODE REVIEW

File: src/app/components/trips/trip-detail/trip-detail.component.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<div [routerLink]="['/user',(userId$ | async)]">
+  Go To User Profile
+</div>
```
Comment (ashish173) [inline src/app/components/trips/trip-detail/trip-detail.component.html]: @zeus999  make it a button or link or something with the name of the traveller, text inside div is very experimental, please don't send a PR with such code.

File: src/app/reducers/trips.reducer.ts [hunk 2]
```diff
@@ -6,69 +7,74 @@ import { Trip } from './../models/trip';
...
 }
 
 const initialState = {
-    ids: [],
-    trips: {},
-    selectedTripId: null,
-    selectedCityId: null
+	trips: {
+		ids: [],
+		trips: {},
```
Comment (ashish173) [inline src/app/reducers/trips.reducer.ts]: Why ids and trips ? I think we are using a wrong data structure, should be something like `{id: trip}` correct me if I am missing something.

Comment (zeus999) [inline src/app/reducers/trips.reducer.ts]: Have a look at at this file **trips-state.ts**. It will make things clear

Comment (ashish173) [inline src/app/reducers/trips.reducer.ts]: Got it, thanks for pointing 👍

**Pull request merged**